github/actions_runners.go

```diff
@@ -45,8 +45,8 @@ func (s *ActionsService) ListRunnerApplicationDownloads(ctx context.Context, own
 	return rads, resp, nil
 }
 
-// GenerateJITConfigRequest specifies body parameters to GenerateJITConfig.
-type GenerateJITConfigRequest struct {
+// GenerateRepoJITConfigRequest specifies body parameters to GenerateRepoJITConfig.
```

Comment:
Since this struct is used for both `GenerateOrgJITConfig` and `GenerateRepoJITConfig`, I think its name should be changed back to `GenerateJITConfigRequest`.